src/pspm_interpolate.m

```diff
@@ -1,51 +1,36 @@
-function [sts, outdata] = pspm_interpolate(varargin)
+function [sts, outdata] = pspm_interpolate(indata, varargin)
 % ● Description
 %   pspm_interpolate interpolates NaN values passed with the indata parameter.
-%   The behaviour of the function can furthermore be adjusted with the
-%   combination of different options.
-%   The function works either on single data sets such as a filename, a
-%   numeric array or a pspm data struct. Alternatively it is possible to pass
-%   a cell containing all these possible datatypes. The function then
-%   iterates through the whole data set and replaces the passed data with the
-%   interpolated data. For filenames the interpolated data will, depending on
-%   option.newfile, be written to the existing file or can also be added to a
-%   new file with filename 'i'+<old filename>. The corresponding cell
-%   (in outdata) will then contain the filename of the new file
-%   (if newfile = 1) or will contain the channel id where the interpolated
-%   data can be found in the existing file (because it has been added or
-%   replaced). The edited data set will then be returned as parameter outdata.
+%   Indata can be a numeric array or a filename. If indata is a filename, 
+%   then the function either acts on one selected channel and 
+%   writes the result to the same file, or on all channels in the file and 
+%   writes a new file if the input is a file name. 
 % ● Format
-%   [sts, outdata] = pspm_interpolate(indata, options)
+%   [sts, outdata] = pspm_interpolate(numeric_array, options)
+%   [sts, outdata] = pspm_interpolate(filename, channel, options)
 % ● Arguments
-%          indata:  [struct/char/numeric] or [cell array of struct/char/numeric]
-%                   contains the data to be interpolated
+%          indata:  [char/numeric] contains the data to be interpolated
```

Comment:
Think the description of `indata` shall be updated to make it inline with above for avoiding confusion...